src/sentry/static/sentry/app/components/releaseProjectStatSparkline.jsx

```diff
@@ -41,21 +46,45 @@ const ReleaseProjectStatSparkline = React.createClass({
         });
       }
     });
+    let issuesPath = `/projects/${orgId}/${projectId}/issues/`;
```

Comment:
so i think since this api is paginated, we don't want to use this endpoint (see https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_group_index.py#L273)

it's probably better to use the project `ProjectReleaseDetailsEndpoint` endpoint for now and use the `new_groups` field from that.